product_supplierinfo_for_customer_sale/models/sale_order_line.py

```diff
@@ -25,38 +27,30 @@ def _compute_product_customer_code(self):
                 code = ""
             line.product_customer_code = code
 
+    def _compute_name(self):
+        """Sobrescribimos el compute para re-definir el name."""
```

Comment:
This comment should be in English and be more descriptive. What you are stating here is the what, which is clear - a method call `_compute_name` does that, to compute the name -, not the why.

Reply:
Sorry, the comment has been changed.